test/system/src/test/java/io/pravega/test/system/ControllerFailoverTest.java

```diff
@@ -134,7 +136,9 @@ public void failoverTest() throws URISyntaxException, InterruptedException {
 
         // Connect with first controller instance.
         URI controllerUri = getTestControllerServiceURI();
-        Controller controller = new ControllerImpl(controllerUri);
+        @Cleanup
+        final Controller controller = new ControllerImpl(controllerUri,
```

Comment:
If we are numbering, then let's use 1 and 2 rather than nothing and 1. Also, if we can qualify rather than number, then better.